lib/rules/prefer-stateless-function/index.js

```diff
@@ -221,8 +223,16 @@ module.exports = {
           property.kind === 'constructor' &&
           isRedundantSuperCall(property.value.body.body, property.value.params)
         ;
+        var isStatic = property.static;
         var isRender = name === 'render';
-        return !isDisplayName && !isPropTypes && !contextTypes && !isUselessConstructor && !isRender;
+        return !(
```

Comment:
i kind of feel like the prior form was more readable; a bunch of "nots" seems easier to understand to me than a bunch of positives, negated